synapseclient/client.py

```diff
@@ -2127,39 +2127,55 @@ def _getUserCredentials(self, url, username=None, password=None):
 
     def createStorageLocationSetting(self, storage_type, **kwargs):
         """
-        Creates a stroage location based on the specified type.
+        Creates an IMMUTABLE storage location based on the specified type.
+
+        For each storage_type, the following kwargs should be specified:
+        ExternalObjectStorage: (S3-like bucket not accessed/managed by synapse)
+         - endpointUrl: endpoint URL of the S3 service (for example: 'https://s3.amazonaws.com')
+         - bucket: the name of the bucket to use
+        ExternalS3Storage: (S3-like bucket accessed/managed by synapse)
```

Comment:
This should be 'S3 bucket access/managed by Synapse', or even better, 'Amazon S3 bucket access/managed by Synapse'.